locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgid ""
+msgstr ""
+"Language: pt-br\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals = 2; plural = (n != 1);\n"
+
+#: src/constants.js:44
+msgid "I want to reset my wallet"
+msgstr "Eu quero resetar minha wallet"
+
+#: src/screens/ChoosePassphrase.js:48
+msgid "To set a blank passphrase mark the corresponding checkbox above."
+msgstr "Marque acima para deixar a frase secreta em branco."
+
+#: src/screens/ChoosePassphrase.js:54
+msgid "Passphrase and confirm passphrase must be equal"
+msgstr "A frase secreta e sua confirmação devem ser iguais"
+
+#: src/components/ModalBackupWords.js:99 src/components/ModalResetAllData.js:41
+#: src/screens/ChoosePassphrase.js:59
+msgid "Invalid password"
+msgstr "Senha inválida"
```

Comment:
Não conferí, mas "password" nesse contexto se refere à mesma coisa que "passphrase"? Em inglês me parece sinônimo (mas não tenho certeza), agora em português "senha" e "frase secreta" soam completamente diferentes.

Reply:
They are not the same. When setting a passphrase we need to write the wallet password and PIN. So this 'Invalid password' references the 'senha

Reply:
Since we're already using "wallet" for "wallet", and since the passphrase is a somewhat advanced feature, maybe we could stick with using "passphrase" in Portuguese too.

Reply:
I translated passphrase as "palavra-secreta" e password as "senha". What do you think?

Reply:
I'm still unsure. I think we can move forward with "palavra-secreta" but create an issue to explore and establish better terms (preferably with precedents). This one in particular I feel like it is and advanced user feature that may end up being harder to understand and search for if we try to use a translated term.

The user either knows or doesn't know the feature:

- if they know, they most likely know the term "passphrase", and "palavra-secreta" (or whatever else) will probably only delay the recognition of the feature;
- if they don't know, it might make sense if we explain really well, but even then they might want to research a little more before using it, and using an uncommon or translated expression will only make the search harder

But I think we shouldn't drag this too much, and I'm completely fine in moving this discussion someplace else.

Reply:
I agree with Jan that some advanced features terms might be easier to understand in english (e.g., passphrase and push tx) but I guess we can continue with it right now without wasting more time and think more about it later.

Reply:
I moved the pt-br translation to PR #74. So, we can merge this one first, and discuss the best translation there.